CMakeLists.txt

```diff
@@ -240,7 +240,7 @@ set(required_deps ASS>=0.15.0
                   Lzo2
                   OpenSSL>=1.1.0
                   PCRE2
-                  RapidJSON>=1.0.2
+                  NlohmannJSON>=3.2.0
```

Comment:
Query, is the 3.2 a hard requirement? Just confirming.

Reply:
Yes, `sax_parse` is available from 3.2.0 which was released in 2018